packages/core/test/eventDescriptors.test.ts

```diff
@@ -60,4 +60,157 @@ describe('event descriptors', () => {
     service.send('NEXT');
     expect(service.state.value).toBe('pass');
   });
+
+  it('should NOT support non-tokenized wildcards', () => {
+    const machine = Machine({
+      initial: 'start',
+      states: {
+        start: {
+          on: {
+            'event*': 'success'
+          }
+        },
+        success: {
+          type: 'final'
+        }
+      }
+    });
+
+    expect(
+      machine.transition(undefined, 'event').matches('success')
+    ).toBeFalsy();
+    expect(
+      machine.transition(undefined, 'eventually').matches('success')
+    ).toBeFalsy();
+  });
+
+  it('should support prefix matching with wildcards (+0)', () => {
+    const machine = Machine({
+      initial: 'start',
+      states: {
+        start: {
+          on: {
+            'event.*': 'success'
+          }
+        },
+        success: {
+          type: 'final'
+        }
+      }
+    });
+
+    expect(
+      machine.transition(undefined, 'event').matches('success')
+    ).toBeTruthy();
+    expect(
+      machine.transition(undefined, 'eventually').matches('success')
+    ).toBeFalsy();
+  });
+
+  it('should support prefix matching with wildcards (+1)', () => {
```

Comment:
the machine from this test is the same as in the previous one - was maybe something else the intention of this test?